UI.AutomationTests/Video/JudgeTests.cs

```diff
@@ -25,6 +25,7 @@ await TestContext.Out.WriteLineAsync(
             // edit display name
             const string newDisplayName = "Edited Judge Name Automated Test";
             judgeWaitingRoomPage.EditJudgeDisplayName(newDisplayName);
+            Thread.Sleep(TimeSpan.FromSeconds(2));
```

Comment:
Why do you need a thread sleep here? Does the `WaitForElementToBeClickable` not work here?

Reply:
it's to to wait for the display name to be updated, as there is a delay before it actually updates. Currently it edits the name and then immediately checks it has the new value, before it has actually updated

Reply:
I see, might be better to get the framework to do it with something like:

Refactor the method `EditJudgeDisplayName` to wait for the updated display name to appear
`
var newNewLocator = By.XPath("//span[@id='host-display-name' and text()='newName']");
WaitForElementToBeVisible(newNewLocator);
`